### ISSUE TITLE
init should not run if a "package.zos.json" already exists

### ISSUE THREAD

**Issue by ajsantander** [zeppelinos/zos-cli#12]
Atm, running init a second time destroys whatever is currently in the file.

**Comment by frangio** [comment]
Agreed. The command should fail if a package.zos.json already exists, and present a message to the user saying so.

If the user wishes to run init anyway, they should delete the file and re-run the command.
